src/styles/_mixins.scss

```diff
@@ -108,3 +108,9 @@
     @include non-visual-button-rules;
   }
 }
+
+@mixin text-lg {
+  font-size: var(--kui-font-size-40, $kui-font-size-40) !important;
+  font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular) !important;
+  line-height: var(--kui-line-height-40, $kui-line-height-40) !important;
+}
```

Comment:
issue: This is a very specific mixin and I'm not sure what the `-lg` prefix has to do with the corresponding font size and line-height. Can you please just include the styles in the component here? 

We aren't using t-shirt sizing

Reply:
Agree, removed